tla-types/src/test/scala/at/forsyte/apalache/tla/typecmp/TestArithmeticBuilder.scala

```diff
@@ -0,0 +1,95 @@
+package at.forsyte.apalache.tla.typecmp
+
+import at.forsyte.apalache.tla.lir._
+import at.forsyte.apalache.tla.lir.oper.TlaArithOper
+
+class TestArithmeticBuilder extends BuilderTest {
+  def testBinaryOperAndBuilderMethod(
+      oper: TlaArithOper,
+      method: (BuilderWrapper, BuilderWrapper) => BuilderWrapper,
+      retT: TlaType1): Unit = {
+    val cmp = sigGen.computationFromSignature(oper)
+
+    val args = Seq.fill(2)(builder.int(1))
```

Comment:
```suggestion
    val args = Seq(builder.int(10), builder.int(20))
```

I don't like riddles in tests :)

Reply:
I'll hold off on these changes, because I'm working on the generalzied testing framework where this should be parameterized by arity anyway.